### PR TITLE
Create alerts-configuration-manager.md

### PR DESCRIPTION
**Pull request by sashwathn**
Documentation Draft for Alerts Configuration Manager from the UI.

### CODE REVIEW

File: docs/cloud/alerts-configuration-manager.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+- Click the `Alert icon` on the top right corner of the chart.
+![Alert Icon](https://github.com/netdata/netdata/assets/96257330/88bb4e86-cbc7-4e01-9c84-6b901188c0de)
+
+- Alert Configuration Modal will open up with `default` thresholds. Modify the Configurations as per need and the alert configuration dialog on the right will be updated dynamically.
+![Alert Configuration Modal](https://github.com/netdata/netdata/assets/96257330/ce39ae64-2ffe-4576-8c92-b7918bb8c91c)
+
+- Use the `Advance Options` for more fine grained definition of the Alerts Configuration
+![Advance Options](https://github.com/netdata/netdata/assets/96257330/b409b31b-6dc7-484c-a2a4-4e5e471d029b)
+
+- Copy the `Alert Configuration` that is generated on the right side of the modal and add it to an existing [health configuration file](https://learn.netdata.cloud/docs/alerting/health-configuration-reference#edit-health-configuration-files) or a new custom file under `<path to netdata install>/etc/netdata/health.d/` on a `Parent Agent` or a `Standalone Child Agent`.
```
Comment (shyamvalsan) [inline docs/cloud/alerts-configuration-manager.md:18]:
```suggestion
- Copy the alert definition that is generated in the code box and add it to an existing [health configuration file](https://learn.netdata.cloud/docs/alerting/health-configuration-reference#edit-health-configuration-files) or a new custom file under `<path to netdata install>/etc/netdata/health.d/` on a `Parent Agent` or a `Standalone Child Agent`.
```

File: docs/cloud/alerts-configuration-manager.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
+# Creating Alerts with Netdata Alerts Configuration Manager
+
+The Netdata Alerts Configuration Manager enables users to create alerts from the Netdata Dashboard with an intuitive user interface.
```
Comment (shyamvalsan) [inline docs/cloud/alerts-configuration-manager.md:3]: @sashwathn should we mention this feature is ONLY available for Netdata business users?

**Review by shyamvalsan — CHANGES_REQUESTED** [review]

File: docs/cloud/alerts-configuration-manager.md (hunk 2 above)
Comment (sashwathn) [reply docs/cloud/alerts-configuration-manager.md:3]: > @sashwathn should we mention this feature is ONLY available for Netdata business users?

Should we include this in the documentation? No hard adding it I guess..

Comment (sashwathn) [reply docs/cloud/alerts-configuration-manager.md]: Done..

File: docs/cloud/alerts-configuration-manager.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
+# Creating Alerts with Netdata Alerts Configuration Manager
+
+The Netdata Alerts Configuration Manager enables users with `Business subscriptions` to create alerts from the Netdata Dashboard with an intuitive user interface.
```
Comment (shyamvalsan) [inline docs/cloud/alerts-configuration-manager.md:3]:
```suggestion
The Netdata Alerts Configuration Manager enables users with `Business subscriptions` to create alerts from the Netdata Dashboard with an intuitive user interface. 
```
```suggestion
The Netdata Alerts Configuration Manager enables users with [Business subscriptions](https://www.netdata.cloud/pricing/) to create alerts from the Netdata Dashboard with an intuitive user interface. 
```

**Review by shyamvalsan — APPROVED** [review]

**Review by shyamvalsan — APPROVED** [review]

**Pull request merged**